components/arm/xarm/xarm.go

```diff
@@ -255,5 +255,25 @@ func (x *xArm) DoCommand(ctx context.Context, cmd map[string]interface{}) (map[s
 	if _, ok := cmd["load"]; ok {
 		return x.getLoad(ctx)
 	}
+	if val, ok := cmd["set"]; ok {
```

Comment:
When this is modularized we should add some good documentation to DoCommand describing to a user how it should be interacted with.

Reply:
My preference is that this simply goes away, and speed/acceleration are specified as part of a time-anchored trajectory.

Reply:
For the time being since we are just using this for a single demo I would prefer to keep this functionality in the DoCommand. 
I could make a ticket to track this improvement though, i.e. removing set_speed & set_acceleration in favor of specifying them "as part of a time-anchored trajectory."
What do you think?

Reply:
This definitely should stay in a DoCommand for this PR. The discussed change above is in the far-off future.

Reply:
Sure.  Its not a super actionable ticket until a bunch of other stuff happens first but I suppose no harm in making it.  Keeping it in DoCommand is definitely the correct scope here

Reply:
[RSDK-8847](https://viam.atlassian.net/browse/RSDK-8847)